src/main/java/org/mskcc/limsrest/controller/GetSampleManifest.java

```diff
@@ -1,10 +1,13 @@
 package org.mskcc.limsrest.controller;
 
+import java.util.Arrays;
```

Comment:
IDE auto-ordered imports, I can remove these changes if desired since only import statement at L8 was actually modified in this PR.